src/Clock.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+type Props = {
+  clockName: number,
+};
+
+type State = {
+  date: Date,
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    date: new Date(),
+  };
+
+  setTime = setInterval(() => {
+    this.setState({ date: new Date() });
+    // eslint-disable-next-line
+    console.log(this.state.date.toLocaleTimeString);
```

Comment:
Something strange in your console right now


https://user-images.githubusercontent.com/66570142/132011247-cd89701f-2cc7-4e2f-b6b1-fddd6133621f.mov